rpm/felix.spec

```diff
@@ -138,7 +125,6 @@ rm -rf $RPM_BUILD_ROOT
 %files -n calico-common
 %defattr(-,root,root,-)
 /usr/bin/calico-diags
-/usr/bin/calico-cleanup
```

Comment:
That was a python tool so it was removed with the clear-out.  Could put it back but I'd want to write a go version rather than maintain the python one.